qiskit/circuit/quantumcircuit.py

```diff
@@ -3115,6 +3116,13 @@ def decompose(
         # do not copy operations, this is done in the conversion with circuit_to_dag
         return dag_to_circuit(dag, copy_operations=False)
 
+    @deprecate_arg(
+        "justify",
+        since="1.1.0",
```

Comment:
Maintainers, do we need to set `pending=True`? I don't know how that works now that we're on Qiskit 1.0.

Reply:
Its says "Set to True if the deprecation is still pending.", and technically we haven't deprecated it yet, so I guess I should add it yes.

Also, just checked, and the rest of @deprecate_arg with `since="1.1.0"` have also `pending=True`.

Reply:
Its says "Set to True if the deprecation is still pending.", and technically we haven't deprecated it yet, so I guess I should add it yes.

Also, just checked, and the rest of `@deprecate_arg` with `since="1.1.0"` have also `pending=True`.

Reply:
Ah, this gets to my other comment I forgot to say: `since` should be `1.2.0` because 1.1.0 is already released. I'm not convinced we want to set `pending=True` because there won't be a 1.3 release, so we wouldn't have the opportunity to switch off `pending` before the 2.0 release, so we could not safely remove this mechanism in 2.0.

This is our justification for `pending`, but it was written before the Qiskit 1.0 deprecation policy, so it might be stale:

> The alternative path must be in place for one minor version before any warnings are issued. For example, if we want to replace the function foo() with bar(), we must make at least one minor release with both functions before issuing any warnings within foo(). You may issue PendingDeprecationWarnings from the old paths immediately, but this is not necessary and does not affect any timelines for removal.
>
> Reason: we need to give people time to swap over without breaking their code as soon as they upgrade.

Let's see what the others think.

Reply:
Changed`"1.1.0"` to `"1.2.0"`.

Reply:
(repeated comment)

----
**Update, from call with luciano and Kevin:**

They think in this case the wrapper complicates things instead of helping, so we should maybe remove it 👍